pandas/tests/frame/test_arithmetic.py

```diff
@@ -72,6 +72,23 @@ def test_tz_aware_scalar_comparison(self, timestamps):
 # -------------------------------------------------------------------
 # Arithmetic
 
+class TestFrameFlexArithmetic(object):
+    def test_df_add_flex_filled_mixed_dtypes(self):
```

Comment:
does this represent a change in user facing API?

Reply:
No, just a case that is not currently tested.  I made a mistake in an early commit in this PR; this test would have caught it.